Assets/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "com.microsoft.mixedrealitytoolkit",
```

Comment:
Is this / should this PR be related to #4616?
Commenting specifically on the name, since there was some discussion either on that or an earlier version on naming conventions

Reply:
I can't find that discussion at the moment though. Maybe @andreiborodin remembers?

Reply:
@keveleigh Good point. This is very very similar to #4616. This PR is intended to allow you to easily bring MRTK pulled down from GitHub into a Unity project. It appears #4616 is about packaging MRTK up as a UPM for distribution, but I _think_ it could work the same way. I will start a question over there and see what comes of it.

Reply:
I have updated the package name based on my comments on #4616 [here](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4616#issuecomment-508151668). Assuming you guys agee with that approach, I believe this is resolved.